### ISSUE TITLE
mysqli::real_connect(): (HY000/2054): Server sent charset (0) unknown to the client.

### ISSUE THREAD

**Issue by felixtech-msp** [phpmyadmin/phpmyadmin#19005]
### Describe the bug

Logging in to phpMyAdmin to MariaDB 11.3.2 is not possible anymore due to the following error message:
![grafik](https://github.com/phpmyadmin/phpmyadmin/assets/89098885/edcaf6a1-5249-4626-a226-2c8e5a74c8f4)

The database charset is utf8mb4 (uca1400_ai_ci), which is the default for the mariadb:latest Docker image.
I am using phpMyAdmin also as the official Docker image.

### Expected behavior

Login is possible as normally.

### Server configuration

- Operating system: VMware PhotonOS
- Web server: as included in the official phpMyAdmin Docker image
- Database version: MariaDB 11.3.2
- PHP version: as included in the official phpMyAdmin Docker image
- phpMyAdmin version: phpmyadmin:latest@sha256:a9695a48170ef0fd8fc64ecb94aac5a3edafb5153ad55cf32407735e029de13e

### Client configuration

- Browser: Mozilla Firefox 113.0
- Operating system: Windows Server 2019

**Comment by kamil-tekiela** [comment]
Can you explain a little bit more what you mean by "anymore"? This is a PHP error and not phpMyAdmin error. I am not sure what we can do about this. This should really be reported to PHP but with a proper MCVE. If you can tell us anything else then maybe we could open an incident on your behalf.

**Comment by kamil-tekiela** [comment]
Actually, it looks like this has already been reported https://github.com/php/php-src/issues/13452 This issue can then be closed probably.

**Comment by felixtech-msp** [comment]
By "anymore" I mean that the day I opened this ticket I was greeted by this error message instead of a working phpMyAdmin. The environment and the database have not changed and I have confirmed the database is indeed working and the charset is correctly set in the database.
We use MariaDB and phpMyAdmin as their official Docker images and so I am unaware of where or what the cause of the issue is.

As the issue occurs only in one of our projects and this is currently in non-production and developers are just always pulling the latest images from DockerHub we don't know in which container version the issue was first introduced.
I'll wait for the upstream fix and for you to creating a Docker image where that is fixed.
Thank you so far.
